doc/faq.md

```diff
@@ -1,160 +1,58 @@
 # Microsoft Azure IoT device SDK FAQ
```

Comment:
@abhipsaMisra I remember we've agreed to delete this. UWP support is partially implemented.